app/src/util/networks.ts

```diff
@@ -352,6 +405,15 @@ export const knownTokens: { [name in KnownToken]: KnownTokenData } = {
     },
     order: 9,
   },
+  eth: {
```

Comment:
adding ETH here adds a second ETH to the token dropdown menu if we haven't loaded the token list from the subgraph list yet:

<img width="294" alt="Screen Shot 2021-01-27 at 11 27 46 am" src="https://user-images.githubusercontent.com/72715177/105924601-b39f8600-6092-11eb-9b8e-0d01bb38324f.png">